### PR TITLE
Compatibility updates for matplotlib 3 and minor tweaks

### PR DESCRIPTION
**Pull request by brovercleveland**

### CODE REVIEW

File: skhep/visual/mpl_plotter.py [hunk 1]
```diff
@@ -691,7 +692,9 @@ def draw_errorbars(self):
         for i in range(n_data_sets_eff):
             if self.err_dict['err_color'] == 'auto' and not self.stacked:
                 if self.histtype == 'marker':
-                    err_color = colors.to_rgba(vis_object[i]._get_rgba_face())
+                    # err_color = colors.to_rgba(vis_object[i]._get_rgba_face())
```
Comment (chrisburr) [inline skhep/visual/mpl_plotter.py]:
```suggestion
```

**Comment by chrisburr** [comment]
I think this PR increases the minimum supported matplotlib version from 2.0.0 to 2.2.0 as `_get_rgba_face` isn't part of the public matplotlib API.

@eduardo-rodrigues Would you rather bump the version in `setup.py` or wrap this in a matplotlib version check?

**Comment by eduardo-rodrigues** [comment]
Hi @chrisburr, that's a good point/question: I would do both, so increate the version number, making it clear in the PR doc that the matplotlib version supported goes up, and I would update setup.py.

It would be good to get a new release out once we sort out PR https://github.com/scikit-hep/scikit-hep/pull/72.

**Comment by brovercleveland** [comment]
This PR removed `_get_rgba_face`, not added it.  As far tests on my end went, this is still compatible with all mpl>2.0.0.

`_get_rgba_face()` was replaced with `(obj.get_markerfacecolor(), obj._alpha)`.

**Pull request merged**